src/MoBi.Presentation/Views/IAddBuildingBlocksToModuleView.cs

```diff
@@ -9,6 +9,7 @@ public interface IAddContentToModuleView<TPresenter> : IBaseModuleContentView<TP
    {
       void ShowInitialConditionsName();
       void DisableDefaultMergeBehavior();
+      void HideDefaultMergeBehaviorGroup();
```

Comment:
Could you rename these ones to not use the term `Default`. There was a time when we were planning to allow the user to override the Module behavior when creating a simulation, but that was removed.